### ISSUE TITLE
Color for whitespace characters

### ISSUE THREAD

**Issue by stevengunneweg** [wayou/vscode-todo-highlight#84]
I configured my editor to show all whitespace characters. When I set the color for a keyword, all whitespace characters also get this color which is very inconvenient. Can you add a variable for whitespace color?

Current:
```
"todohighlight.keywords": [
        {
            "text": "@TODO",
            "overviewRulerColor": "#ffab00",
            "backgroundColor": "#182c42",
            "color": "#ffffff",
            "isWholeLine": true,
        }
]
```

Request:
```
"todohighlight.keywords": [
        {
            "text": "@TODO",
            "overviewRulerColor": "#ffab00",
            "backgroundColor": "#182c42",
            "color": "#ffffff",
            "whitespaceColor": "#636363",
            "isWholeLine": true,
        }
]
```

**Comment by stevengunneweg** [comment]
When i have `isWholeLine: true` and set a color, all characters are colored, including whitespace characters (for example the grey dots in your screenshot). This makes the whole much harder to read.
I would prefer if all whitespace characters would keep their original color and other text is the defined color.

**Comment by wayou** [comment]
how about adding another keyword config for the whitespace
sth like:
```json
{
            "text": " ",
            "color": "#636363"
        },
```

**Comment by stevengunneweg** [comment]
This seems to sort of work but it's a workaround for my problem, not a fix.
This also overrides the default color of the whitespace on lines that don't match the 'text' field.

I tried to find a way to fix this in the sourcecode of the plugin but can't find `isWholeLine` anywhere except the readme. I noticed that there's only one release (v0.5.1) but the version in vscode is v0.5.12. Where can i find the sourcecode of this version?
